doc/11-Concepts.md

```diff
@@ -0,0 +1,77 @@
+![DetailedAnatomy](res/ListItemAnatomyDetailed.jpg "A more detailed list item that spans four lines")
+
+## Overdue Items
+
+In host and service lists overdue checks are now highlighted. This makes it immediately apparent which objects may no longer be up-to-date. The red badge contains information about how long the check has overdue.
+![OverdueListAnatomy](res/ListAnatomyOverdue.jpg "A list with a highlighted item")
+
+## Downtimes
+Downtime lists have an extra information to be displayed: the progress.
+How far along the downtime is, is now immeadiately visible via the progress bar at the top of each list item.
+In the only square visual, we display how much time is left until the downtime ends.
+![DowntimeListItem](res/ListItemDowntimeAnatomy.jpg "A downtime list item with the added progress bar in the top")
+
+
+# Modal
+If you wanted to create a comment or set a downtime from the detail area, the dialog was displayed in another column, which meant that the contents of the left column were lost. In the icingadb module, there is now a modal element for short interaction dialogues. For actions in the detail area, a modal dialog will now appear. This preserves the left list column and thus the context better.
+![ModalAnatomy](res/ModalAnatomy.jpg "A modal presenting a form")
+
+### Utility
+The utility or close button is one if the three ways to close the modal without confirming your changes. You may also just click outside it's borders or press the escape key to escape its' grasp.
+
+### Title
+The title tells you what is asked off you in a simple keyword. Usually an action like: Acknowledge Problem, Add Comment or Schedule Downtime.
+
+### Hint
+The hint explains more about how to interact with the modal and what is asked of you. It will also go into detail what consequences your actions within the modal will have on your environment.
+
+### Form
+The main reason for modals is to perform an action without getting a new tab with your form, which would switch the layout you were working with before.
+
+### Submit
+With the submit button you apply the choses you made in the form and return to where you left off.
+
+# Details
+## Check statistics
+Information about check execution is available in a handy graph that makes it possible to get all of the information at a glance.
+![CheckStatistics](res/CheckStatisticsAnatomy.jpg "A panel with the statistics about check execution")
+
+### Descriptions and Values
+Instead of having a table with the different check information there is now a header that lays out the information in an intuitive way, giving more weight to more important bits of data and their values.
+
+### Graph
+The graphical element in the middle makes it easier to see, whether the check is on time or late. It's easy to immediately see the configured check intervall and have a better grasp on how up-to-date the current state is.
+
+### Indicator
+The indicator visualises where the "now" is for orientation.
```

Comment:
Maybe rephrase this to `The indicator marks the current date and time for better orientation.`.